deploy/FullDeployTest.js

```diff
@@ -125,6 +125,24 @@ module.exports = async function (hre) {
         `Gas Price Oracle Adapter Deployed ${gasPriceOracleAdapter.address}`
     )
 
+    const fastGasOracle = await deploy("FastGasOracle", {
+        from: deployer,
+        log: true,
+        args: [ethOracle.address, gasOracle.address],
+        contract: "GasOracle",
+    })
+
+    const fastGasOracleAdapter = await deploy("GasPriceOracleAdapter", {
+        from: deployer,
+        log: true,
+        args: [fastGasOracle.address],
+        contract: "OracleAdapter",
+    })
+
+    console.log(
+        `Fast Gas Oracle Adapter Deployed ${gasPriceOracleAdapter.address}`
+    )
+
```

Comment:
As per the above comment, https://github.com/tracer-protocol/tracer-protocol/blob/contracts-v2/deploy/FullDeployTest.js#L55 already deploys a gas oracle